spec/suspenders/generators/runner_generator_spec.rb

```diff
@@ -1,24 +1,76 @@
 require "spec_helper"
 
 RSpec.describe Suspenders::RunnerGenerator, type: :generator do
-  it "generates and destroys the configuration for the app to run" do
-    with_fake_app do
-      copy_file "bin_setup", "bin/setup"
-      copy_file "README.md.erb", "README.md"
+  before_generate = proc do
```

Comment:
That's a good point! The reason is that this setup doesn't relate to most expectations here, so it is perceived as noise to me. In the case you pointed out, the information is relevant to the test, so it's explicit - we're removing `spec_helper.rb` and creating `test_helper.rb`, which is crucial to make the minitest assertion work.

However, I don't feel strongly and I could use local helper methods as you noted! It would add one more line to each test but that's no big deal.